Alerting/Sample Watches/ml_examples/default_ml_watch.json

```diff
@@ -0,0 +1,220 @@
+                  }
+                },
+                "aggs": {
+                  "top_influencer_hits": {
+                    "top_hits": {
+                      "sort": [
+                        {
+                          "influencer_score": {
+                            "order": "desc"
+                          }
+                        }
+                      ],
+                      "_source": {
+                        "includes": [
+                          "result_type",
+                          "timestamp",
+                          "influencer_field_name",
+                          "influencer_field_value",
+                          "influencer_score",
+                          "isInterim"
+                        ]
+                      },
+                      "size": 3,
+                      "script_fields": {
+                        "score": {
+                          "script": {
+                            "lang": "painless",
+                            "source": """Math.round(doc["influencer_score"].value)"""
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "record_results": {
+                "filter": {
+                  "range": {
+                    "record_score": {
+                      "gte": 3
+                    }
+                  }
+                },
+                "aggs": {
+                  "top_record_hits": {
+                    "top_hits": {
+                      "sort": [
+                        {
+                          "record_score": {
+                            "order": "desc"
+                          }
+                        }
+                      ],
+                      "_source": {
+                        "includes": [
+                          "result_type",
+                          "timestamp",
+                          "record_score",
+                          "is_interim",
+                          "function",
+                          "field_name",
+                          "by_field_value",
+                          "over_field_value",
+                          "partition_field_value"
+                        ]
+                      },
+                      "size": 3,
+                      "script_fields": {
+                        "score": {
+                          "script": {
+                            "lang": "painless",
+                            "source": """Math.round(doc["record_score"].value)"""
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "condition": {
+      "compare": {
+        "ctx.payload.aggregations.bucket_results.doc_count": {
+          "gt": 0
+        }
+      }
+    },
+    "actions": {
```

Comment:
I think there was an email action for the default Watch. Can you please dbl check and include if it is available. This included the link back to the anomaly explorer which is useful.